src/sys/Net/NetServices.cs

```diff
@@ -555,6 +555,7 @@ public static IPAddress GetLocalAddressForRemote(IPAddress destination)
                     {
                         // Socket exception is thrown if the OS cannot find a suitable entry in the routing table.
                     }
+                    udpClient.Close();

```

Comment:
Release the port

Reply:
The UdpClient will be cleaned up automatically, since it goes out of scope, but it's not a bad idea to explicitly close and avoid the need to wait for the garbage collector. An even better way would be to put the UdpClient in a "using" scope so that it gets closed and disposed of.

Reply:
Thanks for feedback. I don't know when the UdpClient is cleaned up after going out of scope (waiting GC). I found the port is still in use for a long time until I kill the process (in windows 10, not sure in Linux).
I put it in "using" scope and it could be closed as expected. Thanks for the suggestion!